pkg/networkservice/chains/nsmgr/server.go

```diff
@@ -83,8 +77,9 @@ var _ Nsmgr = (*nsmgrServer)(nil)
 func NewServer(ctx context.Context, nsmRegistration *registryapi.NetworkServiceEndpoint, authzServer networkservice.NetworkServiceServer, tokenGenerator token.GeneratorFunc, registryCC grpc.ClientConnInterface, clientDialOptions ...grpc.DialOption) Nsmgr {
 	rv := &nsmgrServer{}
 
-	var urlsRegistryServer registryapi.NetworkServiceEndpointRegistryServer
-	var localbypassRegistryServer registryapi.NetworkServiceEndpointRegistryServer
+	var localBypassRegistryServer, interposeRegistryServer registryapi.NetworkServiceEndpointRegistryServer
+
+	filterMechanismsServer := filtermechanisms.NewServer(nsmRegistration.Url, &localBypassRegistryServer)
```

Comment:
why this outside of networkservice chain

Reply:
https://github.com/networkservicemesh/sdk/pull/698#discussion_r571854769